pkg/cmd/openshift-sdn-node/kube_proxy.go

```diff
@@ -314,5 +313,5 @@ func startProxyServer(s *ProxyServer) error {
 
 	go s.Proxier.SyncLoop()
 
-	return <-errCh
+	return nil
```

Comment:
you changed this in one commit and then changed it back in the next

Reply:
yeah I thought I should fix this but it turns out it was valid if we have suppot bindaddress hardfail so I revert it back.